src/ansi-c/library/string.c

```diff
@@ -500,7 +516,7 @@ inline char *strdup(const char *str)
 
 #undef memcpy
 
-inline void *memcpy(void *dst, const void *src, size_t n)
+void *memcpy(void *dst, const void *src, size_t n)
```

Comment:
A benefit of the 'inline' is that the assertions get the location of the call site.